### PR TITLE
Dev/create kwiver wheel

### PR DESCRIPTION
**Pull request by as6520**
Kwiver wheel including sprokit and vital. At the moment it does not support opencv processes or processes that would require packages other than pybind11, boost and eigen

Depends on #877 #867 #865

### CODE REVIEW

File: python/kwiver/vital/tests/test_config.py [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+        get_valuea = nc.get_value(keyb)
+        nose.tools.assert_equal(valuea, get_valuea)
+
+    @nose.tools.raises(BaseException)
+    def test_get_value_no_exist(self):
+        c = config.empty_config()
+        keya = 'keya'
+        keyb = 'keyb'
+        valueb = 'valueb'
+        c.get_value(keya)
```
[the remark this answers is not in the document]
Comment (as6520) [reply python/kwiver/vital/tests/test_config.py]: Good catch, I forgot to rebase #877 after fixing this issue in #848

**Comment by Erotemic** [comment]
I'm starting to test the wheel. I'll add comments as I see them: 

* `kwiver.__version__` doesn't exist, can we add that?

* Can I get a python 3.7 wheel, are we building wheels with manylinux? Note: I have a repo that builds manylinux wheels here: https://github.com/Erotemic/hesaff/tree/master if you need a reference.

**Comment by as6520** [comment]
> * Can I get a python 3.7 wheel

I'll create a python 3.7 wheel and PM you the location. 

> * are we building wheels with manylinux?

I don't believe we can build manywheel linux at the moment since we link against python library to register python plugins. This violates the external libraries that we can link against as specified in [manylinux wheel policy](https://www.python.org/dev/peps/pep-0571/#the-manylinux2010-policy).

**Comment by Erotemic** [comment]
Can auditwheel fix that issue?

**Comment by as6520** [comment]
Maybe but it would copy the python library inside the wheel to remove this dependency. This might cause problems if the python interpreter is using a different python library then what we are providing in the wheel. Although we might need to experiment to verify if this issue arises.

I had thought about using pybind11::scoped_interpreter_guard{} instead of loading the python symbols manually. pybind11 does not specify libpython dependency when interpretor guard is used. This might remove the dependency but I didn't try it in this PR.

**Comment by Erotemic** [comment]
Its likely best to just get some sort of wheel out there first, and then address the manylinux issue in a separate PR. I think the pybind11::scoped_interpreter_guard sounds like it might work. There is also the option that we build a wheel that is manylinux complaint in every respect except for linking against the python library. I think in most cases that wouldn't cause a problem.

**Comment by as6520** [comment]
Closing since the changes have been merged in dev/overhaul branch

**Pull request closed without merging**